src/MahMaterialDragablzMashUp/MainWindow.xaml

```diff
@@ -36,6 +36,7 @@
           <Button Content="Hello World" />
           <Button Content="Nice Popup" />
           <Button Content="Goodbye" />
+          <Button Content="Version" />
```

Comment:
Not sure if this should also be a button since there is no functionality behind it, but I guess it's so this fits in with the rest of the items (Buttons) above